src/constants/endpoints.ts

```diff
@@ -7,7 +7,9 @@ export interface DisciplineType {
   icon: string
   numberOfChildren: number
   isLeaf: boolean
+  isFirstLevel: boolean
   apiKey?: string
+  total_training_sets?: number
```

Comment:
why have you added this? We have the logic to map the total_training_sets-Number to the numberOfChildren.

Reply:
See below. If you like that better I could also introduce a new property `hasTrainingSets` or `needsTrainingSetsEndpoint` or similar

Reply:
i see, why the previous usage of `isLeaf` is confusing (and inconsistent with our appview of disciplines), but i'm really unhappy to add this property. 
Would it be a better solution to add a property `childrenType`, which can be `'discipline' || 'training_set || 'documents'`, just a thought, not sure if this would make anything better...
But at least i would prefer one of your suggested boolean-variable, as there is no need to pass the numberOfChilderen twice.

Reply:
can you please rename? I would prefer `needsTrainsSetsEndpoint: boolean`

Reply:
Done